app/lang/en.json

```diff
@@ -42,7 +45,7 @@
     "enterDateOfBirth": "Enter date of birth",
     "dateOfBirthMustBeReal": "Date of birth must be a real date",
     "dateOfBirthIsFuture": "Date of birth must be before today",
-    "dateOfBirthIsUnder18": "You are under 18. I understand that I must be at least 18 on the date I sign the LPA, otherwise the LPA will be rejected.",
+    "dateOfBirthIsUnder18": "This attorney is under 18 years old. You will not be able to finish making this LPA until they are 18.",
```

Comment:
this content is also used on `/your-details`

Reply:
Good catch, will break out into two keys